src/Apps/CollectorProfile/Components/CollectorProfileHeaderInfo.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  BriefcaseIcon,
+  Flex,
+  IconProps,
+  MapPinIcon,
+  MuseumIcon,
+  Text,
+} from "@artsy/palette"
+import { FC, Fragment } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Media } from "Utils/Responsive"
+import { CollectorProfileHeaderInfo_me$data } from "__generated__/CollectorProfileHeaderInfo_me.graphql"
+
+const M_ICON_SIZE = 14
+const D_ICON_SIZE = 18
```

Comment:
**suggestion:** Maybe we can give this a better name. I'm not sure if I understand intuitively what `M_` and `D_` means.